sources/packages/backend/libs/integrations/src/sfas-integration/sfas-files/sfas-part-time-application-record.ts

```diff
@@ -48,4 +48,45 @@ export class SFASPartTimeApplicationRecord extends SFASRecordIdentification {
       ? +this.line.substring(49, 49 + 10).trim()
       : null;
   }
+  /**
+   * CSPT Award Amount (sail_extract_data.sail_cspt_award_amt).
+   */
+  get CSPTAward(): number | null {
+    return this.line.substring(59, 59 + 10).trim()
```

Comment:
Do we require null check for award fields?? I know that existing award fields are doing null check. Please check if it is required.